### ISSUE TITLE
[Adaptive Cards - ExpenseReport]: Talkback is announcing twice role information as Link and Button for the 'Export as PDF' link.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#201]
### Target Platforms

Android

### SDK Version

1.0.1 (2)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 1.0.1(2)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'ExpenseReport'.
8. Navigate to 'Export as PDF' link and Observe the issue.

**Actual Result:**
While navigating to the 'Expense Report' card, Talkback is announcing twice role information as 'Link and button'. It is announcing as 'Export as PDF link button.'

**Expected Result:**
Talkback should announce correct role information as link for the expense report. It should announce as **'Export as pdf link.'**

**Note:**
Same issue is observed for the 'Reply Link' under Input screen.

**User Impact:**
It will impact the talkback user as they will get confused if the screen reader will announce the twice role for any control also due to this user will not able whether on activating the control what action will be performed.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/a5004480-76e2-43fd-ac5e-cfb288a99dd9

![image](https://github.com/user-attachments/assets/983057a7-1f41-4336-8027-1c2751fb5219)

### Card JSON

```JSON
NA
```
